endpoints/events/event.go

```diff
@@ -185,15 +186,28 @@ func HandleAccountServiceErrors(errs []error) (status int, messages []string) {
 	messages = []string{}
 	status = http.StatusInternalServerError
 
+	//grab error messages
 	for _, er := range errs {
-		errCode := errortypes.ReadCode(er)
+		messages = append(messages, er.Error())
+	}
 
+	//set appropriate error code
+	for _, er := range errs {
+		if errors.Is(er, context.DeadlineExceeded) {
```

Comment:
This is good with me. It's new functionality in Go 1.13, but that's our minimum supported version.